CardIO_Public_API/CardIOPaymentViewController.h

```diff
@@ -76,6 +76,13 @@
 /// Defaults to NO.
 @property(nonatomic, assign, readwrite) BOOL suppressScannedCardImage;
 
+/// After a successful scan, the CardIOView will briefly display an image of the card with
```

Comment:
Oh, 1 more teensy picky request. We shouldn't mention `the CardIOView` here. If you can think of a better wording, go for it. Or else you can leave that to me, post-merge.